.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    webextensions: true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:import/warnings",
+  ],
+  globals: {
+    ChromeUtils: false,
+    ExtensionAPI: false,
+    // NOTE: These get injected via Rollup.
+    __API_ENDPOINT__: false,
+    __ION_STUDIES_LIST__: false,
+    __ION_WEBSITE_URL__: false,
+    __STORE_IMPLEMENTATION__: false,
+  },
+  overrides: [
+    {
+      files: "tests/**",
+      env: {
+        mocha: true,
+      },
+      extends: [
+        "plugin:mocha/recommended",
```

Comment:
Added mocha rules to the tests/* directory since it flagged a few "no-async-describe" usages.

```
/Volumes/Dev/github/mozilla/ion-core-addon/tests/core-addon/Storage.test.js
   6:5   error  'sinon' is assigned a value but never used  no-unused-vars
  15:25  error  Unexpected async function in describe()     mocha/no-async-describe
  25:38  error  Unexpected async function in describe()     mocha/no-async-describe
  71:38  error  Unexpected async function in describe()     mocha/no-async-describe
```